app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -261,7 +283,22 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
               download(values.id, file?.name ?? `${values.source}-${values.id}`);
             }}
           />
+          <Button
+            onClick={() => {
+              setMedia();
+            }}
+            style={{ marginTop: '2 px', marginLeft: '10px' }}
```

Comment:
Don't do inline styling

Reply:
I was following the existing approach...but perhaps that's Jonny's code? There are 7 examples of inline styling in ContentSummaryForm.

Reply:
Jonny has added inline styling, which I have asked him to refrain from.  Some of it has slipped through due to time constraints.

Reply:
I applied styling using styled.